libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -339,6 +339,11 @@ class YulUtilFunctions
 	/// Reverts on invalid input.
 	/// signature: (addr) -> value
 	std::string readFromCalldata(Type const& _type);
+	/// @returns a function that reads a value type from stack address. Performs
```

Comment:
We won't have stack addresses in Yul, only variables.
The stack is allocated and managed by LLVM.